test/SessionManagerTest.php

```diff
@@ -902,6 +904,29 @@ public function testIdValidationWillFailOnInvalidData(): void
         $this->manager->start();
     }
 
+    #[RunInSeparateProcess]
```

Comment:
It's not actually necessary to add this test, there are already tests covering it:

https://github.com/laminas/laminas-session/blob/2.23.x/test/SessionManagerTest.php#L542-L562

This test will pass regardless of whether you've reverted the lines in question or not.

Reply:
There is actually no test, checking that the session is not marked as started after setting a session id

Reply:
OK - the test will still pass though, regardless of whether those lines in SessionManager have been changed or not.

Can you verify that reverting those lines fixes your problem, and do you have any idea if you're doing something "non-standard" that is causing the problem? If you can isolate that, we can have a new test that fails before, and passes after this change.